src/more_speech/ui/swing/article_panel.clj

```diff
@@ -1,17 +1,24 @@
 (ns more-speech.ui.swing.article-panel
-  (:require [more-speech.ui.swing.edit-window :as edit-window]
-            [more-speech.ui.swing.ui-context :refer :all]
-            [more-speech.ui.swing.article-tree-util :as article-tree-util]
-            [more-speech.config :as config]
-            [more-speech.ui.formatters :as formatters]
+  (:require [more-speech.config :as config]
             [more-speech.nostr.events :as events]
             [more-speech.nostr.util :as util]
-            [more-speech.ui.swing.util :as swing-util])
-  (:use [seesaw core border])
-  )
+            [more-speech.ui.formatters :as formatters]
+            [more-speech.ui.swing.article-tree-util :as article-tree-util]
+            [more-speech.ui.swing.edit-window :as edit-window]
+            [more-speech.ui.swing.ui-context :refer :all]
+            [more-speech.ui.swing.util :as swing-util :refer [copy-to-clipboard]])
+  (:use [seesaw core border]))
 
 (declare id-click bold-label)
 
+(defn copy-click [e]
+  (when (.isPopupTrigger e)
+    (let [x (.x (.getPoint e))
+          y (.y (.getPoint e))
+          node (.getComponent e)
+          p (popup :items [(action :name "Copy" :handler #(copy-to-clipboard (.getText node) %))])]
```

Comment:
Do you see this `%`, I haven't found information on how to remove it, I mean the extra argument which is passed to this fn. If don't type it explicit error occurs about the arguments arity of this fn.

Reply:
The :handler functions of actions take the event that caused the action.  I usually use partial as a way to avoid the # and % in :handlers.  See the changes I made in my latest push.

Thanks for the update.  That's a big help.  Feel free to help more!

Reply:
I mean that, I don't need this argument(event) passed by event emitter at all, because I'm not using it :(
But can't find out how to create function wich will ignore it in clojure, for example in js it would work:
```
const emitEvent = (handlerFn) => return handlerFn(event); // expect handlerFn to take AT LEAST one arg
const handlerFn = () => do();//
emitEvent(handlerFn); // wil work in js/ equal in clojure will throw arity error, because handlerFn should accept one argument but actually is accepting 0 args  
```

My pleasure, I will for sure :)